pkgs/top-level/python-aliases.nix

```diff
@@ -90,6 +90,7 @@ mapAliases ({
   hdlparse = throw "hdlparse has been removed, it was using setuptools 2to3 translation feature, which has been removed in setuptools 58"; # added 2022-01-18
   hyperkitty = throw "Please use pkgs.mailmanPackages.hyperkitty"; # added 2022-04-29
   IMAPClient = imapclient; # added 2021-10-28
+  imdbpy = throw "imdbpy has been renamed to cinemagoer"; # added 2022-08-08
```

Comment:
Why not
```suggestion
  imdbpy = cinamagoer; # added 2022-08-08
```
?

Reply:
> Why not
> ?

Probably it's a good idea to keep the throw: If you consider that [there are 17 packages still depending on the deprecated name](https://libraries.io/pypi/IMDbPY/dependents) and that https://pypi.org/project/imdbpy is not capable of doing the redirect alone, this throw here may help people to consider pushing the updates upstream. Who knows how many end-users scripts are also using the outdated package...

Reply:
> What do you want to be done?

Nothing, I just wondered about the reasoning.

Reply:
> Nothing, I just wondered about the reasoning.

Didn't seem necessary. (A combination of: No references. Upstream had moved on. Niche package. Warning was added still.)